docs/website/views/_layouts/popular-articles.njk

```diff
@@ -5,11 +5,19 @@ layout: content
 <h2 class="popular-articles-title">Welcome on Exadel authoring Kit for AEM</h2>
 <h3 class="popular-articles-subheader">Popular articles</h3>
 <ul class="popular-articles-container">
-  {% for article in popularArticles %}
-    <li class="popular-articles-item">
-      <h4 class="mb-2">{{ article.title }}</h4>
-      <p>{{ article.textPreview }}</p>
-      <a href="{{ article.url | url }}">Learn More</a>
-    </li>
+  {% set title = r/<h2.*?>.*?<\/h2>/ig %}
+  {% set content = r/<p.*>.*?<\/p>/ig %}
+  {% for item in collections.all %}
+    {% for link in articlesLink %}
```

Comment:
Use the filter possibility of njk, then you have more control over the optimization